address_book/src/lib.rs

```diff
@@ -32,31 +31,101 @@ static LOGGING: Lazy<()> = Lazy::new(|| unsafe {
     tide::log::with_level(LOG_LEVEL);
 });
 
+// TODO: remove test store after tests, add TestFileStore type that implements Drop?
```

Comment:
@nyospe I tried to do this but it turned out pretty messy because I didn't find a working way to organize the traits and structs.

Do you know how I could have a version of the Store for tests that implements `Drop` and deletes the store path when inside `drop`? Or any other way to clean up more or less reliably after the tests. At least the files should not remain if the tests fail.

Reply:
This is already built into `tempdir::TempDir`. I'm just not using it the right way at the moment.